nominal/core/data_review.py

```diff
@@ -96,9 +96,8 @@ def archive(self) -> None:
     @property
     def nominal_url(self) -> str:
         """Returns a link to the page for this Data Review in the Nominal app"""
-        # TODO (drake): move logic into _from_conjure() factory function to accomodate different URL schemes
         run = self._clients.run.get_run(self._clients.auth_header, self.run_rid)
-        return f"https://app.gov.nominal.io/runs/{run.run_number}/?tab=checklists&openChecklistDetails={self.rid}&openCheckExecutionErrorReview="
+        return f"{self._clients.app_base_url}/runs/{run.run_number}/?tab=checklists&openChecklistDetails={self.rid}&openCheckExecutionErrorReview="  # noqa: E501
```

Comment:
this is very annoying, the line length increased by _two characters_ but then failed E501 as "line length 148 > 120"
🤷🏼